releases/release-1.29/release-notes/maps/pr-119946-map.yaml

```diff
@@ -1,5 +1,5 @@
 pr: 119946
 releasenote:
-  text: '`kubeadm`: turn on feature gate `MergeCLIArgumentsWithConfig` to merge the
-    config from flag and config file, otherwise, if the flag `--ignore-preflight-errors`
+  text: '`kubeadm`: turned on feature gate `MergeCLIArgumentsWithConfig` to merge
```

Comment:
text: '`kubeadm`: will now turn on feature gate `MergeCLIArgumentsWithConfig` to merge the config from flag and config file, otherwise. If the flag `--ignore-preflight-errors` is set from the CLI, then the value from config file will be ignored.'

Reply:
With "turn" I think it works better as is: "will now" is especially useful for things that describe a change of behaviour ("show more logs", since "showed more logs" is actually incorrect).